config/initializers/secret_token.rb

```diff
@@ -4,4 +4,4 @@
 # If you change this key, all old signed cookies will become invalid!
 # Make sure the secret is at least 30 characters and all random,
 # no regular words or you'll be exposed to dictionary attacks.
-ReviewSite::Application.config.secret_token = 'ce6de5cf6ab565296342ab430b7db186ad49405984140b85e6ebebcd37fd803e70d90c6074160492b6c0f25e913dd7a42824a324ef564edd0dca991c9824c620'
+ReviewSite::Application.config.secret_token = ENV['RAILS_SECRET_TOKEN']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.